ci/Jenkinsfile.android

```diff
@@ -82,6 +87,18 @@ pipeline {
         }
       }
     }
+    stage('Check APK Size') {
+      steps {
+        script {
+          def maxSizeInMB = params.ALLOWED_APK_SIZE.toInteger()
+          def apkSize = sh(script: "du -m ${apks[0]} | cut -f1", returnStdout: true).trim().toInteger()
+          echo "APK size: ${apkSize} MB"
+          if (apkSize > maxSizeInMB) {
+            error "APK size (${apkSize} MB) exceeds the maximum allowed size of ${maxSizeInMB} MB"
```

Comment:
This is the wrong place for such checks. Please add it to the `apk` library and use `assert` for this:
https://github.com/status-im/status-jenkins-lib/blob/ab9aa93c13eb42fb524b2f8e54521c73da140c9c/vars/apk.groovy#L49-L70

Reply:
Ah nice! I will move this to `status-jenkins-lib` 
Reasoning for making this a parameter would be to relax these settings in future if we needed to..

Reply:
https://github.com/status-im/status-jenkins-lib/pull/94/commits/e97837713689924e62d41321bce5e756ee3a748e